db_service/setup_db.go

```diff
@@ -35,6 +35,11 @@ func SetupDb(logger lager.Logger) *gorm.DB {
 	dbHost := os.Getenv("DB_HOST")
 	dbUsername := os.Getenv("DB_USERNAME")
 	dbPassword := os.Getenv("DB_PASSWORD")
+
+	if dbPassword == "" || dbHost == "" || dbUsername == "" {
```

Comment:
this change is probably suited for it's own cleanup CL, it's a positive change but it's not related